### ISSUE TITLE
Support mobile browsers

### ISSUE THREAD

**Issue by mzgol** [github/fetch#182]
There is nothing in README or in [.travis.yml](https://github.com/github/fetch/blob/master/.travis.yml) about mobile browser support. Is something planned?

I've also noticed the Travis config mentions only IE 9-11 & Chrome; what about Firefox?

Thanks.

**Comment by dgraham** [comment]
The polyfill should work fine in mobile browsers. However, we just use `XMLHttpRequest` on github.com mobile because of the large `Promise` polyfill dependency that's required to use `fetch`.

This [Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1172246) prevents the tests from running.

**Comment by mzgol** [comment]
> However, we just use `XMLHttpRequest` on github.com mobile because of the large `Promise` polyfill dependency that's required to use `fetch`.

I understand; the question is if you mostly treat this project as one just for your (GitHub) needs that's open source if anyone wants to look or as a more general `window.fetch` polyfill - in the latter case testing at least on iOS/Android would be worthwhile.

If someone e.g. uses a custom jQuery compilation for both desktop & mobile sites, compiling away `jQuery.ajax` and using `jQuery.Deferred` as a `Promise` polyfill wouldn't increase the size too much compared to bare `jQuery` (and could even decrease it, I didn't check). This will be possible in jQuery 3.0 which will have Promises/A+-compliant Deferred implementation.

Disclaimer: I'm a jQuery Core Team member (I'm writing privately here, though); some time ago we considered creating a simpler `jQuery.xhr` interface as an alternative to `jQuery.ajax`; we resigned mostly because of the upcoming `window.fetch` and polyfills like this one; we figured it would be better to direct people to [github/fetch](https://github.com/github/fetch) than create a completely new interface from scratch.

Thanks for the Bugzilla link.
